### PR TITLE
check for start of the path, trim if start with "/" or "\" when setting the destination folder.

### PR DESCRIPTION
**Pull request by YangLyu89**
This is to resolve https://github.com/aspnet/LibraryManager/issues/307

### CODE REVIEW

File: src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ internal partial class InstallDialog : DialogWindow
 
         public InstallDialog(IDependencies dependencies, ILibraryCommandService libraryCommandService, string configFileName, string fullPath, string rootFolder)
         {
-            InstallationFolder.DestinationFolder = fullPath.Replace(rootFolder, "").Replace('\\', '/');
```
Comment (jodavis) [inline src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs]: >Replace [](start = 60, length = 7)

Use StringComparison.OrdinalIgnoreCase here

File: src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs [hunk 2]
```diff
@@ -24,7 +24,7 @@ internal partial class InstallDialog : DialogWindow
...
+            InstallationFolder.DestinationFolder = fullPath.Replace(rootFolder, "").TrimStart(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar).Replace('\\', '/');
```
Comment (jodavis) [inline src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs]: >fullPath.Replace(rootFolder, "").TrimStart(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar).Replace('\\', '/') [](start = 51, length = 122)

nit: Break this into multiple lines to make it easier to read.

File: src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs (hunk 1 above)
Comment (YangLyu89) [reply src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs]: use starswith for it

---
In reply to: [202842715](https://github.com/aspnet/LibraryManager/pull/316#discussion_r202842715) [](ancestors = 202842715)

File: src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs (hunk 2 above)
Comment (YangLyu89) [reply src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs]: Will do

---
In reply to: [202842892](https://github.com/aspnet/LibraryManager/pull/316#discussion_r202842892) [](ancestors = 202842892)

**Pull request merged**